app/javascript/packs/sdk.js

```diff
@@ -12,7 +12,7 @@ import {
 } from '../sdk/cookieHelpers';
 import { addClasses, removeClasses } from '../sdk/DOMHelpers';
 import { SDK_SET_BUBBLE_VISIBILITY } from 'shared/constants/sharedFrameEvents';
-const runSDK = ({ baseUrl, websiteToken, referral }) => {
+const runSDK = ({ baseUrl, websiteToken, referral, ref }) => {
```

Comment:
Does referral and ref have the same value? wdyt @Arjona99

Reply:
Yep, `referral` has the same purpose of `ref` variable @cmm-apli , so there is no need to receive a new param

Reply:
done